lib/active_interactor/interactor/worker.rb

```diff
@@ -60,10 +60,16 @@ def execute_context!
       end
 
       def execute_context_with_callbacks!
-        interactor.run_callbacks :perform do
+        result = interactor.run_callbacks :perform do
           execute_context_with_validation_check!
           @context = interactor.finalize_context!
         end
+
+        if context&.success? && interactor.respond_to?(:run_after_perform_callbacks_on_children)
+          interactor.run_after_perform_callbacks_on_children
+        end
```

Comment:
@aaronmallen if I leave the callbacks in are you okay with a PR that includes just this change to move where the deferred callbacks get executed?